python/lsst/pipe/tasks/processImage.py

```diff
@@ -240,68 +185,15 @@ def matchSources(self, exposure, sources):
         - matches  list of reference object/source matches (an lsst.afw.table.ReferenceMatchVector)
         - matchMeta  metadata about the field (an lsst.daf.base.PropertyList)
         """
-        try:
-            astrometry = self.calibrate.astrometry
-        except Exception:
-            self.log.warn("Failed to find an astrometry solver in the calibrate task")
-            return None, None
-
-        astromRet = astrometry.loadAndMatch(exposure=exposure, sourceCat=sources)
+        astromRet = self.getAstrometer().loadAndMatch(exposure=exposure, sourceCat=sources)
+        # N.b. yes, this is what useKnownWcs calls the returned values
```

Comment:
But we're not calling anything called useKnownWcs here; it seems like that's an implementation detail of ANetBasicAstrometryTask, but we're not obviously using that either.